yktrace/src/lib.rs

```diff
@@ -13,10 +13,7 @@
 extern crate test;
 
 use core::yk_swt::SirLoc;
-use std::{
```

Comment:
This hunk is due to a rustfmt change. I wasn't sure if it should be in a separate commit or not.